ERCS/eip-draft_smart_blob.md

```diff
@@ -0,0 +1,269 @@
+---
```

Comment:
```suggestion
---
eip: 7689
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.